ports/restbed/portfile.cmake

```
@@ -1,28 +1,29 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
-set(USE_OPENSSL OFF)
-if("openssl" IN_LIST FEATURES)
-    set(ADDITIONAL_PATCH "add_openssl_support.patch")
-    set(USE_OPENSSL ON)
-endif()
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    openssl     BUILD_SSL 
+)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Corvusoft/restbed
-    REF 0f8af8d8ed183a88e208adeb22da0080d5d74d1e
-    SHA512 f0175a10c88f1ad4f16c8e4cff7ceea7b80c56b0724b59791c23e91f1ecf146dfdbda9e9238d31a35f21d8cdcc413b586cc633725dd0ba87de6b599a7087916f
+    REF 4.7
+    SHA512 f8aaa89ae5c862253f9a480fefa8827927e3a6c13c51938bbcd6c28ac4e6d54496ecf51610343e9a3efe3e969314f643a487506c606a65f56125b51ab8478652
     HEAD_REF master
     PATCHES
-        cmake.patch
-        ${ADDITIONAL_PATCH}
+        use-FindOpenSSL-cmake.patch
+        asio-1-18-0-deprecations.patch

```

```suggestion
        asio-1-18-0-deprecations.patch #https://github.com/Corvusoft/restbed/pull/444/files
```
It would be better to add this comment here, so we can remove this patch in next release easily.